common/src/main/java/com/microsoft/identity/common/IPrtCredentialHolder.kt

```diff
@@ -0,0 +1,16 @@
+package com.microsoft.identity.common;
+
+import android.app.Activity
+
+/**
+ * Consumer of commons needs to implement [IPrtCredentialHolder] interface
+ * and set it using CommonPrtCredentialHolder.initializeCommonPrtCredentialHolder(@NonNull IPrtCredentialHolder prtCredentialHolder)
+ * to provide prtCredentialHolder to common module.
+ */
+interface IPrtCredentialHolder {
```

Comment:
This is called a holder but doesn't seem to be holding anything. To me this looks more like a controller.